Hit Airtable directly

allthesignals opened the pull request:
This commit uses the API for a “public” user in Airtable, who has been given read-only access to the table.

Addresses #6

allthesignals committed:
Hit Airtable directly

This commit uses the API for a “public” user in Airtable, who has been given read-only access to the table.

Addresses #6;

chriswhong commented:
Recommend we not add jQuery if all we need is ajax calls.  Stick with a lighter weight package like `fetch`

allthesignals committed:
Use fetch instead of jQuery

Avoid jQuery dependency since it is better for DOM manipulation. Use Fetch instead.
